### ISSUE TITLE
Changelog out of date

### ISSUE THREAD

**Issue by veniosg** [openintents/filemanager#95]
The changelog shown on launch after update / first use seems out of date by some versions.

**Comment by friedger** [comment]
Any ideas how to better manage that?

**Comment by veniosg** [comment]
Not really, it would have to be automated to ensure that it doesn't get missed, but changelogs have to be created manually anyway. 
I would be in favour of removing the changelog altogether seeing as most stores nowadays offer this and github tags should be enough for anyone needing historic or more granular logs.
